app/v3/controllers/SubmitSelfEmploymentBsasController.scala

```diff
@@ -122,22 +120,26 @@ class SubmitSelfEmploymentBsasController @Inject()(val authService: EnrolmentsAu
     }
 
   private def errorResult(errorWrapper: ErrorWrapper) = {
+    // @formatter:off
+    
     (errorWrapper.error: @unchecked) match {
-      case BadRequestError | NinoFormatError | CalculationIdFormatError |
-           CustomMtdError(RuleIncorrectOrEmptyBodyError.code) | RuleBothExpensesError |
-           CustomMtdError(FormatAdjustmentValueError.code) | RuleTypeOfBusinessIncorrectError |
-           CustomMtdError(RuleAdjustmentRangeInvalid.code) => BadRequest(Json.toJson(errorWrapper))
-      case RuleSummaryStatusInvalid | RuleSummaryStatusSuperseded |
-           RuleAlreadyAdjusted | RuleOverConsolidatedExpensesThreshold |
-           RuleTradingIncomeAllowanceClaimed | RuleNotSelfEmployment |
-           RuleResultingValueNotPermitted => Forbidden(Json.toJson(errorWrapper))
+      case BadRequestError | NinoFormatError | CalculationIdFormatError | CustomMtdError(RuleIncorrectOrEmptyBodyError.code) |
+          CustomMtdError(RuleBothExpensesError.code) |
+          CustomMtdError(FormatAdjustmentValueError.code) | RuleTypeOfBusinessIncorrectError | CustomMtdError(RuleAdjustmentRangeInvalid.code) =>
+        BadRequest(Json.toJson(errorWrapper))
+
+      case RuleSummaryStatusInvalid | RuleSummaryStatusSuperseded | RuleAlreadyAdjusted | RuleOverConsolidatedExpensesThreshold |
+          RuleTradingIncomeAllowanceClaimed | RuleNotSelfEmployment | RuleResultingValueNotPermitted =>
```

Comment:
`RuleNotSelfEmployment` is deprecated and  was replaced with `RuleTypeOfBusinessIncorrectError`  you can take it out